### PR TITLE
Increase default `body-parser` limit for standalone server

### PR DESCRIPTION
**Pull request by trevor-scheer**
In `apollo-server`, the default limit for the `body-parser` package was `50mb`. This restores the previous limit and adds a test to confirm it remains at least > the default (100KiB).

### CODE REVIEW

File: packages/server/src/__tests__/standalone/standaloneSpecific.test.ts [hunk 1]
```diff
@@ -103,3 +103,41 @@ describe('Typings: TContext inference', () => {
...
+      },
+    });
+
+    const { url } = await startStandaloneServer(server, {
+      listen: { port: 0 },
+    });
+
+    const excessivelyLargeBody = JSON.stringify({
+      query: `{hello}`,
+      variables: Object.fromEntries([...Array(7960)].map((_, i) => [i, 'foo'])),
```
Comment (glasser) [inline packages/server/src/__tests__/standalone/standaloneSpecific.test.ts:124]: GraphQL doesn't require all variable values to match declared variables, but it's maybe a little awkward that these variables don't even have valid names (they start with a digit)...  I mean, not "wrong" but you could also make it actually valid pretty easily (can just be one very long string or one very long array of ints too).

**Review by glasser — CHANGES_REQUESTED** [review]
Needs a changeset.

We also need to update the doc that tells you how to convert startStandaloneServer to expressMiddleware, and potentially also update other calls to body-parser in the docs.

File: .changeset/stale-ligers-impress.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@apollo/server': patch
+---
+
+Restore body-parser limit of standalone server to 50mb (as it was in the `apollo-server` package)
```
Comment (glasser) [inline .changeset/stale-ligers-impress.md:5]:
```suggestion
startStandaloneServer: Restore body-parser request limit to 50mb (as it was in the `apollo-server` package in Apollo Server 3)
```

**Comment by trevor-scheer** [comment]
@glasser my intuition is _not_ to recommend the 50mb limit configuration (users should probably only tweak as needed?) in all of our docs examples, but just make note of the default in the "swapping" article as you mentioned.

File: packages/server/src/__tests__/standalone/standaloneSpecific.test.ts (hunk 1 above)
Comment (trevor-scheer) [reply packages/server/src/__tests__/standalone/standaloneSpecific.test.ts]: [93d24a4](https://github.com/apollographql/apollo-server/pull/7172/commits/93d24a408438da81e1d5088fd79d54f3024063f7)

**Review by glasser — APPROVED** [review]

**Pull request merged**